repository/src/main/java/com/epam/esm/repository/impl/query/NewSqlQuery.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.esm.repository.impl.query;
+
+public class NewSqlQuery {
```

Comment:
Use enums for constants, and use descriptive and meaningful names for classes.

Reply:
class renamed. Can you explain why I cannot use the class with private constructor and public static final fields instead of enum?